Help/Hackathon.rst

```diff
@@ -0,0 +1,206 @@
+Hackathon Page
+==============
+
+This page intends to help the organisation of local hackathons. To start,
```

Comment:
Maybe you should clarify that this page is for the people in the hackthon, and not the organizers?